Automattic/wp-calypso
# Jetpack Cloud: Add History Items

#### Changes proposed in this Pull Request
This PR adds Historic Threats that are fixed or ignored by using the new QueryScanHistory 
component. 

I think a lot of this data that comes from the api still needs clean up. 

After:
Placeholder:
<img width="1088" alt="Screen Shot 2020-04-17 at 2 09 53 PM" src="https://user-images.githubusercontent.com/115071/79567878-3da6d100-80b5-11ea-88df-940066212bf9.png">

With data from the api.
<img width="1375" alt="Screen Shot 2020-04-17 at 2 07 22 PM" src="https://user-images.githubusercontent.com/115071/79567697-f6204500-80b4-11ea-98f2-02f820298d53.png">

#### Testing instructions

Load This PR and visit 
* http://jetpack.cloud.localhost:3000/scan/history/example.com
* Notice that the treats info shows up as expected.
